### PR TITLE
[All] Update to compose 1.0.0-alpha07

### PR DESCRIPTION
**Pull request by florina-muntenescu**

### CODE REVIEW

File: BasicsCodelab/build.gradle [hunk 1]
```diff
@@ -17,21 +17,24 @@
...
     dependencies {
-        classpath 'com.android.tools.build:gradle:4.2.0-alpha15'
+        classpath 'com.android.tools.build:gradle:4.2.0-alpha16'
         classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
     }
 }
 
 allprojects {
     repositories {
+        maven {
```
Comment (chrisbanes) [inline BasicsCodelab/build.gradle:35]: Can delete this

File: LayoutsCodelab/build.gradle [hunk 2]
```diff
@@ -35,6 +35,9 @@ buildscript {
 
 allprojects {
     repositories {
+        maven {
```
Comment (chrisbanes) [inline LayoutsCodelab/build.gradle:38]: Can delete

**Review by nickbutcher — APPROVED** [review]

File: BasicsCodelab/build.gradle [hunk 3]
```diff
@@ -17,21 +17,24 @@
...
+            url "https://androidx.dev/snapshots/builds/6961312/artifacts/repository/"
```
Comment (manuelvicnt) [inline BasicsCodelab/build.gradle:36]: Remove

File: MigrationCodelab/build.gradle [hunk 4]
```diff
@@ -18,23 +18,23 @@ buildscript {
...
-        compileSdkVersion = 28
+        compileSdkVersion = 30
         minSdkVersion = 21
-        targetSdkVersion = 28
+        targetSdkVersion = 30
 
         // App dependencies
         appCompatVersion = '1.1.0'
-        composeVersion = '1.0.0-alpha06'
+        composeVersion = '1.0.0-SNAPSHOT'
```
Comment (chrisbanes) [inline MigrationCodelab/build.gradle:27]: alpha07?

File: LayoutsCodelab/build.gradle [hunk 5]
```diff
@@ -35,6 +35,9 @@ buildscript {
...
+            url "https://androidx.dev/snapshots/builds/6961312/artifacts/repository/"
```
Comment (manuelvicnt) [inline LayoutsCodelab/build.gradle:39]: Remove

File: StateCodelab/build.gradle [hunk 6]
```diff
@@ -17,21 +17,25 @@
...
     dependencies {
-        classpath "com.android.tools.build:gradle:4.2.0-alpha15"
+        classpath "com.android.tools.build:gradle:4.2.0-alpha16"
         classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
     }
 }
 
 allprojects {
     repositories {
+        maven {
```
Comment (chrisbanes) [inline StateCodelab/build.gradle:35]: delete

File: ThemingCodelab/build.gradle [hunk 7]
```diff
@@ -16,21 +16,24 @@
...
     dependencies {
-        classpath "com.android.tools.build:gradle:4.2.0-alpha15"
+        classpath "com.android.tools.build:gradle:4.2.0-alpha16"
         classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
     }
 }
 
 allprojects {
     repositories {
+        maven {
```
Comment (chrisbanes) [inline ThemingCodelab/build.gradle:34]: delete

File: MigrationCodelab/app/build.gradle [hunk 8]
```diff
@@ -87,7 +93,8 @@ dependencies {
     implementation "androidx.compose.material:material:$rootProject.composeVersion"
     implementation "androidx.compose.runtime:runtime-livedata:$rootProject.composeVersion"
     implementation "androidx.ui:ui-tooling:$rootProject.composeVersion"
-    implementation "com.google.android.material:compose-theme-adapter:1.0.0-alpha06"
+//    implementation "com.google.android.material:compose-theme-adapter:$rootProject.composeVersion"
```
Comment (manuelvicnt) [inline MigrationCodelab/app/build.gradle:96]:
```suggestion
    implementation "com.google.android.material:compose-theme-adapter:$rootProject.composeVersion"
```

File: MigrationCodelab/app/build.gradle [hunk 9]
```diff
@@ -87,7 +93,8 @@ dependencies {
...
+    implementation 'com.github.material-components:material-components-android-compose-theme-adapter:dev_alpha07-SNAPSHOT'
```
Comment (manuelvicnt) [inline MigrationCodelab/app/build.gradle:97]: Remove this line, not needed if the one above is uncommented

File: MigrationCodelab/build.gradle (hunk 4 above)
Comment (manuelvicnt) [inline MigrationCodelab/build.gradle:27]:
```suggestion
        composeVersion = '1.0.0-alpha07'
```

File: MigrationCodelab/build.gradle [hunk 10]
```diff
@@ -68,8 +68,23 @@ plugins {
 
 allprojects {
     repositories {
+        maven { // Compose alpha07 snapshot
+            url "https://androidx.dev/snapshots/builds/6953474/artifacts/repository/"
```
Comment (manuelvicnt) [inline MigrationCodelab/build.gradle:72]: Remove this maven repo and the jitpack one below

File: MigrationCodelab/build.gradle [hunk 11]
```diff
@@ -68,8 +68,23 @@ plugins {
...
+        }
         google()
         jcenter()
+        maven { url 'https://jitpack.io' } // Needed for Compose's MDC Theme adapter
+    }
+}
+
+subprojects {
+    configurations.configureEach {
+        resolutionStrategy.eachDependency { details ->
```
Comment (manuelvicnt) [inline MigrationCodelab/build.gradle:82]: Remove this block

File: StateCodelab/build.gradle [hunk 12]
```diff
@@ -17,21 +17,25 @@
...
+            def snapshot = "6953474"
```
Comment (manuelvicnt) [inline StateCodelab/build.gradle:36]: Remove

File: ThemingCodelab/build.gradle [hunk 13]
```diff
@@ -16,21 +16,24 @@
...
+            url "https://androidx.dev/snapshots/builds/6961312/artifacts/repository/"
```
Comment (manuelvicnt) [inline ThemingCodelab/build.gradle:35]: Remove

**Review by manuelvicnt — CHANGES_REQUESTED** [review]

**Review by manuelvicnt — APPROVED** [review]
Two nits: Remove the `.DS_Store` file and check with Nick if the run configurations should be removed

**Review by nickbutcher — CHANGES_REQUESTED** [review]
Good catch Manu, run configurations shouldn't be deleted.

**Review by nickbutcher — APPROVED** [review]

**Review by objcode — APPROVED** [review]

**Pull request merged**